scripts/createS3Bucket.js

```diff
@@ -0,0 +1,28 @@
+const aws = require('aws-sdk');
+const s3 = new aws.S3();
+
+const bucketName = process.env.AWS_SERVICE_NAME;
+
+s3.listBuckets((err, data) => {
+  if (err) {
+    console.log('Error', err);
+    process.exit(1);
+  }
+
+  if (data.Buckets.find(bucket => bucket.Name === bucketName)) {
+    console.log(`Bucket "${bucketName}" already exists.`);
+  } else {
+    const bucketParams = {
+      Bucket: bucketName
+    };
+
+    s3.createBucket(bucketParams, (err, data) => {
+      if (err) {
+        console.log(`Bucket "${bucketName}" failed to build.`, err);
+        process.exit(1);
+      } else {
+        console.log(`Bucket "${bucketName}" built in region ${data.Location}`);
+      }
+    });
+  }
+});
```

Comment:
Bit of a pain, but please can you refactor this to use `async`/`await`?

Broadly something like:

```js
const aws = require('aws-sdk');

const s3 = new aws.S3();
const bucketName = process.env.AWS_SERVICE_NAME;

async function main() {
  const { Buckets } = await s3.listBuckets().promise();

  if (Buckets.find(bucket => bucket.Name === bucketName)) {
    console.log(`Bucket "${bucketName}" already exists.`);
  } else {
    const bucketParams = {
      Bucket: bucketName
    };

    await s3.createBucket(bucketParams).promise();
    console.log(`Bucket "${bucketName}" built in region ${data.Location}`);
  }
});

main().catch(err => {
  console.log('Error', err);
  process.exit(1);
});
```